models/api.model.js

```diff
@@ -1,6 +1,28 @@
 const db = require('../db/connection');
 const endpointsJSON = require("../endpoints.json");
 
+selectArticleById = (article_id) => {
+    return db
+    .query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .then((result) => {
+        if (typeof Number(article_id) !== "number"){
+            console.log("not a number")
+            return Promise.reject({
+                status: 400,
+                msg: "Bad Request"
+            })
+        }
+        else if (result.rows.length === 0) {
+            console.log("Article_ID not found")
+            return Promise.reject({
+                status: 404,
+                msg: "ID Not found"
+            })
+        } else {
+            console.log("ID found")
```

Comment:
Console.logs are really only to help us debug, but once we finish our code (and if we put our code up for code review we are essentially saying, this is a finished piece of work that can be used like this in production and merged into main) these aren't needed any more. Do try and get rid of them once you don't need them any more. 

The ones in the app.js I sort of understand, as you might still want them for the later tasks, however you don't really need them here anymore :)